larray/tests/test_session.py

```diff
@@ -672,15 +665,15 @@ def test_create_typedsession_instance(meta):
 
     # load from file
     ts = TestConstrainedSession(inputpath('test_session.h5'))
-    assert ts.names == ['a', 'a01', 'a2', 'ano01', 'anonymous', 'b', 'b024', 'e', 'f', 'g', 'h']
+    assert ts.names == ['a', 'a01', 'a2', 'ano01', 'anonymous', 'b', 'b024', 'c', 'e', 'f', 'g', 'h']
```

Comment:
all those ts variables look odd now that the class is no longer named typesession